ci/build_container/build_container.sh

```diff
@@ -84,13 +84,12 @@ cd ..
 rm -fr cares* c-ares*
 
 # protobuf
-wget https://github.com/google/protobuf/releases/download/v3.0.0/protobuf-cpp-3.0.0.tar.gz
-tar xf protobuf-cpp-3.0.0.tar.gz
-cd protobuf-3.0.0
+wget https://github.com/google/protobuf/releases/download/v3.2.0/protobuf-cpp-3.2.0.tar.gz
+tar xf protobuf-cpp-3.2.0.tar.gz
+cd protobuf-3.2.0
 ./configure --prefix=$THIRDPARTY_BUILD --enable-shared=no
 make install
 cd ..
-rm -fr protobuf*
```

Comment:
I'm not sure it really matters that much at this point, but this is going to leave a bunch of object files around that will make the image larger. Is there any easy way to deal with this?